.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Kogibot CI
+
+on: pull_request
+jobs:
+  test:
+    name: Build&Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Test with Maven
+      run: mvn clean install
+  
+  checkstyle:
+    name: Validate formatting
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
```

Comment:
why not java 11 and 17?

Reply:
For now focusing on Java 11. We don't need to have many version... Depends which one we will use in the end